Source/Android/app/src/main/res/layout/list_item_mapping.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout android:id="@+id/root"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:attr/selectableItemBackground"
+    android:clickable="true"
+    android:focusable="true"
+    android:nextFocusRight="@id/button_more_settings">
+
+    <TextView
+        android:id="@+id/text_setting_name"
+        style="@style/TextAppearance.MaterialComponents.Headline5"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginEnd="@dimen/spacing_large"
+        android:layout_marginStart="@dimen/spacing_large"
+        android:layout_marginTop="@dimen/spacing_large"
+        android:textSize="16sp"
+        android:textAlignment="viewStart"
+        android:layout_toStartOf="@+id/button_more_settings"
+        tools:text="Setting Name" />
+
+    <TextView
+        android:id="@+id/text_setting_description"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentStart="true"
+        android:layout_alignStart="@+id/text_setting_name"
+        android:layout_below="@+id/text_setting_name"
+        android:layout_marginBottom="@dimen/spacing_large"
+        android:layout_marginEnd="@dimen/spacing_large"
+        android:layout_marginStart="@dimen/spacing_large"
+        android:layout_marginTop="@dimen/spacing_small"
+        android:layout_toStartOf="@+id/button_more_settings"
+        android:textAlignment="viewStart"
+        tools:text="@string/overclock_enable_description" />
+
+    <Button
+        android:id="@+id/button_advanced_settings"
```

Comment:
```suggestion
        android:id="@+id/button_more_settings"
```

Change this to match the id set in the root layout. Make sure you change this for the rtl layout too.

Reply:
I'm not sure what you mean by the root layout, but I noticed I hadn't updated the `layout_toStartOf` fields. Is that what you meant?

Reply:
To clarify, I intentionally made the change from `button_more_settings` to `button_advanced_settings` because:

1. The purpose of the button is a little different (as I wanted to indicate by having different icons for them)
2. In the future, I'm probably going to end up having to have both of these buttons on the Extension setting, since that setting both is an input expression and hides a submenu behind it

Reply:
I meant the field for `nextFocusRight`. Thinking back, "root layout" is probably the wrong term to use here. I should have said "root element" or something more explicit. You fixed it all though so it works perfect now